src/components/atoms/IconAtom/IconAtom.jsx

```diff
@@ -0,0 +1,11 @@
+//We need to import svg sprite here
+
+function IconAtom({ id, className }) {
+  return (
+    <svg className={className}>
+      <use href={sprite + '#icon-' + id}></use>
+    </svg>
+  );
+}
+
+export { IconAtom };
```

Comment:
What do you think about implementing separated logic that will choose and put right class name? 🤔

Reply:
✅